app/Http/Controllers/JobController.php

```diff
@@ -67,12 +67,29 @@ public function managerIndex()
             ->withCount('submitted_applications')
             ->get();
 
+
+        /*
+            Check if job has a title. Append both bool (and french title if locale is 'en').
+            Because global fallback-locale set to true (config/translatable.php) any french translatables are set to the english copy. 'getTranslationsArray()' will retreive real value of translations.
+        */
+        $locale = app()->getLocale();
+        foreach ($jobs as &$job) {
```

Comment:
What does the ampersand do here?

Reply:
https://www.php.net/manual/en/language.references.pass.php